Samples/MultiWindowSampleApplication/ViewModels/LoginViewModel.cs

```diff
@@ -1,25 +1,25 @@
 ﻿using MultiWindowSampleApplication.Models;
 using SimpleWPF.Input;
 using SimpleWPF.ViewModels;
+using System.Threading.Tasks;
 using System.Windows.Input;
 using TestingApplication.Models.Cache;
 
 namespace MultiWindowSampleApplication
 {
     public class LoginViewModel : NavigationViewModelBase
     {
-        public ICommand LoginCommand { get; set; }
-
+        public IAsyncCommand LoginCommand { get; set; }
         public LoginViewModel()
         {
-            LoginCommand = new RelayCommand(Login);
+            LoginCommand = new AsyncCommand(async (parameter) => await Login());
```

Comment:
Any reason why `LoginCommand = new AsyncCommand(x => Login());` wasn't used?

Reply:
Just to simulate the _Login Process_, like there's some asynchronous stuff going on, also to test _AsyncCommand_ and see its status

Reply:
oops, there's no reason. I was in hurry, sorry.